src/liballoc/tests/btree/map.rs

```diff
@@ -269,6 +284,12 @@ fn test_iter_mut_mutation() {
 
 #[test]
 fn test_values_mut() {
+    let mut a: BTreeMap<_, _> = (0..2).map(|i| (i, i)).collect();
+    test_all_refs(&mut 13, a.values_mut());
```

Comment:
Should we also test this is a 144-element tree (or whatever the threshold was for getting another layer of nodes)?

Reply:
Definitely seems like a good idea to me (it's 12 to open up the second layer).